packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -700,18 +707,23 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         legendColor: this.state.color,
         shouldHighlight,
       });
-      const barHeight: number = Math.max(yBarScale(point.y), 0);
-      let adjustedBarHeight = 0;
-
-      if (barHeight <= 0) {
+      const barHeight: number = this.props.supportNegativeYValues
+        ? yBarScale(point.y) - yBarScale(0)
+        : yBarScale(point.y);
```

Comment:
why remove math.max.

Reply:
Math.max function was necessary before because the chart didn't handle negative values. When negative values were encountered, it would force the bar height to 0, ensuring that no bars went below the baseline.
But now new logic explicitly handles the negative values by adjusting the scale calculation.